Error when running solr-export-statistics command

## Describe the bug
When running solr-export-statistics command on DSpace CLI, the following exception happens:

```
org.dspace.util.SolrImportExportException: Could not read start of month batch as date: 2025-08-01T00:00:00Z
	at org.dspace.util.SolrImportExport.exportIndex(SolrImportExport.java:561)
	at org.dspace.util.SolrImportExport.main(SolrImportExport.java:148)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:569)
	at org.dspace.app.launcher.ScriptLauncher.runOneCommand(ScriptLauncher.java:284)
	at org.dspace.app.launcher.ScriptLauncher.handleScript(ScriptLauncher.java:135)
	at org.dspace.app.launcher.ScriptLauncher.main(ScriptLauncher.java:100)
Caused by: java.time.format.DateTimeParseException: Text '2025-08-01T00:00:00Z' could not be parsed, unparsed text found at index 7
	at java.base/java.time.format.DateTimeFormatter.parseResolved0(DateTimeFormatter.java:2055)
	at java.base/java.time.format.DateTimeFormatter.parse(DateTimeFormatter.java:1954)
	at java.base/java.time.YearMonth.parse(YearMonth.java:297)
	at java.base/java.time.YearMonth.parse(YearMonth.java:282)
	at org.dspace.util.SolrImportExport.exportIndex(SolrImportExport.java:559)
	... 8 more
```

## To Reproduce
Using DSpace 9.1 and Solr 9.8.1, just run `dspace solr-export-statistics` with no additional params.

## Expected behavior
Statistics CSV files should be written to the default export dir ([dspace.dir]/solr-export).

## Related work
I belive the bug appeared after [these changes](https://github.com/DSpace/DSpace/commit/3accb4d7d6b48698ca911282d660d596d2b3e505#diff-b6d5ef0debc84fce6d6a4e15357da526ff57194ba7717e03dcbb6e123f1e4407).

It seems that `YearMonth.parse` with only one argument expects a default string format of `uuuu-MM` (see [YearMonth docs](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/YearMonth.html#parse(java.lang.CharSequence))), while facet ranges coming from Solr are ISO date and time.

I will provide a PR with a fix next week.

--- a/dspace-api/src/main/java/org/dspace/util/SolrImportExport.java
+++ b/dspace-api/src/main/java/org/dspace/util/SolrImportExport.java
@@ -556,7 +556,7 @@ public class SolrImportExport {
             YearMonth monthStartDate;
             String monthStart = monthFacet.getValue();
             try {
-                monthStartDate = YearMonth.parse(monthStart);
+                monthStartDate = YearMonth.parse(monthStart, DateTimeFormatter.ISO_OFFSET_DATE_TIME);
             } catch (DateTimeParseException e) {
                 throw new SolrImportExportException("Could not read start of month batch as date: " + monthStart, e);
             }

